services/meta/client.go

```diff
@@ -100,6 +100,12 @@ func (c *Client) Close() error {
 	return nil
 }
 
+func (c *Client) Data() *Data {
```

Comment:
you're right, it shouldn't. But then I'll have to rename either the function or the field to something other than data. Thoughts?